certified-connectors/Ordnance Survey Places/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2985 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "OS Places",

```

Comment:
@quickjc1 The title `"OS Places"` looks generic name, Can you please rename it to `Ordnance Survey's Places` ?

Reply:
@sriyen-msft agreed, will update